projects/nae-example-app/src/app/doc/tabbed-case-view/tabbed-case-view/tabbed-case-view.component.scss

```diff
@@ -1,16 +1,16 @@
 .case-view-container {
-    margin: 16px;
+    margin: 8px;
     height: calc(100% - 32px);
     overflow: auto
 }
 
 .case-view-search-container {
-    padding: 8px 16px;
+    padding: 8px 8px;
 }
 
 .case-header-padding {
-    padding-bottom: 16px;
-    padding-top: 16px;
+    padding-bottom: 8px;
+    padding-top: 8px;
```

Comment:
why are these styles only changed in the example app?

Reply:
done